Week4/app/src/main/java/com/em4n0101/mymovies/MoviesViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.em4n0101.mymovies
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import com.em4n0101.mymovies.utils.Utilities
+
+class MoviesViewModel(application: Application) : AndroidViewModel(application) {
+    private val context = application.applicationContext
```

Comment:
You don't want to have `Context` in the ViewModel class because it gets harder to test. I know we are not doing tests right now, but we will talk about that in the upcoming sessions.